config/exp/etcdrestore/kustomization.yaml

```diff
@@ -0,0 +1,35 @@
+# This kustomization.yaml is not intended to be run by itself,
+# since it depends on service name and namespace that are out of this kustomize package.
+# It should be run by config/default
+resources:
+- crd/bases/turtles-capi.cattle.io_etcdmachinesnapshots.yaml
+- crd/bases/turtles-capi.cattle.io_etcdsnapshotrestores.yaml
+#+kubebuilder:scaffold:crdkustomizeresource
+
+patches:
+# [WEBHOOK] To enable webhook, uncomment all the sections with [WEBHOOK] prefix.
+# patches here are for enabling the conversion webhook for each CRD
+- target:
+    group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+    name: etcdmachinesnapshots.turtles-capi.cattle.io
+  path: crd/patches/turtles-capi.cattle.io_etcdmachinesnapshots.yaml
+- target:
+    group: apiextensions.k8s.io
+    version: v1
+    kind: CustomResourceDefinition
+    name: capiproviders.turtles-capi.cattle.io
```

Comment:
this needs a change found by @yiannistri https://github.com/rancher/turtles/pull/506#discussion_r1586502725